sklearn/multioutput.py

```diff
@@ -22,10 +22,15 @@
 from .base import BaseEstimator, clone, MetaEstimatorMixin
 from .base import RegressorMixin, ClassifierMixin, is_classifier
 from .model_selection import cross_val_predict
+from .utils import check_random_state, _print_elapsed_time
 from .utils.metaestimators import available_if
-from .utils import check_random_state
-from .utils.validation import check_is_fitted, has_fit_parameter, _check_fit_params
 from .utils.multiclass import check_classification_targets
+from .utils.validation import (
+    check_is_fitted,
+    has_fit_parameter,
+    _check_fit_params,
+    _deprecate_positional_args,
```

Comment:
There is a [linting](https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=45077&view=logs&j=32e2e1bb-a28f-5b18-6cfc-3f01273f5609&t=8a54543f-0728-5134-6642-bedd98e03dd0) issue since `_deprecate_positional_args` is not used anymore.